ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -140,12 +140,12 @@
   GOQUORUM_V_VALUE(-50100, "Signature v value not 37 or 38 for GoQuorum private transaction."),
   GET_PRIVATE_TRANSACTION_NONCE_ERROR(-50100, "Unable to determine nonce for account in group."),
   OFFCHAIN_PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "Offchain Privacy group does not exist."),
-  ONCHAIN_PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "Onchain Privacy group does not exist."),
-  ONCHAIN_PRIVACY_GROUP_NOT_ENABLED(-50100, "Onchain privacy groups not enabled."),
+  FLEXIBLE_PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "Flexible Privacy group does not exist."),
+  FLEXIBLE_PRIVACY_GROUP_NOT_ENABLED(-50100, "Flexible privacy groups not enabled."),
   OFFCHAIN_PRIVACY_GROUP_NOT_ENABLED(
-      -50100, "Offchain privacy group can't be used with Onchain privacy groups enabled."),
-  ONCHAIN_PRIVACY_GROUP_ID_NOT_AVAILABLE(
-      -50100, "Private transactions to onchain privacy groups must use privacyGroupId"),
+      -50100, "Offchain privacy group can't be used with Flexible privacy groups enabled."),
+  FLEXIBLE_PRIVACY_GROUP_ID_NOT_AVAILABLE(
+      -50100, "Private Marker Transactions to flexible privacy groups must use privacyGroupId"),
```

Comment:
this should be "Private transactions"
Line 151 should say 
      "Privacy Marker Transaction failed due to intrinsic gas exceeding the limit. Gas limit used from the Private Transaction."),
- small change but it's unrelated to your PR - up to you if you sneak it in.

Reply:
4b9192a